### DISCUSSION TITLE
Booloader dump

### DISCUSSION THREAD

**Discussion by Thatindianbruh** [moonglow/FlashForge_Marlin#129] [Q&A] [answered]
Does anyone have the bootloader from a dremel 3d20? I ended up bricking mine yesterday. 

If you have a bootloader dump for any other printer. Drop it anyway, might help someone else. 

Thanks

**Comment by moonglow** [comment, marked as answer]
Original bootlader without board hash patch from another board will be not works with yours...  You no need to have original bootloader, as example you can try that one: https://github.com/moonglow/FlashForge_Marlin/discussions/101#discussioncomment-2356978
